src/components/Form.test.js

```diff
@@ -13,191 +13,175 @@ describe('Form', () => {
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'Hamburg')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('Hamburg')
+        expect(startSearch).toHaveBeenCalledWith('Hamburg')
     })
 
     it('calls function with valid zip code', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, '22303')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('22303')
+        expect(startSearch).toHaveBeenCalledWith('22303')
     })
 
     it('calls function with valid special characters', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'Rotenburg-Wümme')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledWith('Rotenburg-Wümme')
+        expect(startSearch).toHaveBeenCalledWith('Rotenburg-Wümme')
     })
 
     it('doesnt call function with one or less characters', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'A')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledTimes(0)
+        expect(startSearch).toHaveBeenCalledTimes(0)
     })
 
     it('doesnt call function with 33 or more characters', () => {
-        const setUserPlace = jest.fn()
+        const startSearch = jest.fn()
 
         const { getByPlaceholderText, getByText } = render(
-            <Form setUserPlace={setUserPlace} />
+            <Form startSearch={startSearch} />
         )
         const input = getByPlaceholderText(/Ort oder PLZ eingeben.../)
         const button = getByText(/Suchen/)
 
         user.type(input, 'ABCDEFGHIJKLMNOPQRSTUVWXYZ-ABCDEF')
         user.click(button)
 
-        expect(setUserPlace).toHaveBeenCalledTimes(0)
+        expect(startSearch).toHaveBeenCalledTimes(0)
     })
 
     it('doesnt call function with invalid number (1)', () => {
```

Comment:
Would indicate what the difference between (1) and (2) is. Not obvious on first sight.
For example: 
 - " it does not call function when zip code is too short/long"
 - " it does not call function when zip code has more/less than five digits"
 
 Edit for all tests: -> which function should not be called? Maybe it should be named as well, in order to make it clear.